CHANGELOG.md

```diff
@@ -77,6 +128,16 @@ INTERNAL:
 
 * Load embedded provider schemas for providers found in stacks files into state ([#1763](https://github.com/hashicorp/terraform-ls/issues/1763))
 
+## 0.34.3 (03 September 2024)
+
+ENHANCEMENTS:
+
+* Generate built in providers ([#1817](https://github.com/hashicorp/terraform-ls/issues/1817))
+
+INTERNAL:
+
+* Add artifacts manifest ([#1784](https://github.com/hashicorp/terraform-ls/issues/1784))
+
```

Comment:
It looks like the 0.34.3 was misplaced when I did the last release. Unfortunately changie orders on version number, not date, so this looks out of order.